lib/events/dynamoevents/dynamoevents.go

```diff
@@ -758,7 +773,12 @@ func (l *Log) searchEventsRaw(fromUTC, toUTC time.Time, namespace string, eventT
 	} else {
 		left = math.MaxInt64
 	}
-	doFilter := len(eventTypes) > 0
+
+	var typeFilter *string
+	if len(eventTypes) != 0 {
+		typeList := eventFilterList(len(eventTypes))
```

Comment:
The reason we create variables and define them instead of baking the event type into the string is to avoid injection attacks. With this way Dynamo parses the query first and then fills in the variables we define.